src/modules/addcomment.js

```diff
@@ -0,0 +1,9 @@
+const addcomment = (Obj, key) => fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments`, {
+  method: 'POST',
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+  body: JSON.stringify(Obj),
+}).then((res) => res)
+  .then((output) => output);
+export default addcomment;
```

Comment:
- [ ] According to the project requirement,  you should Use `async` and `await` with the `fetch` method to send and receive data but I can see that you used Promises. Kindly avoid this
- Syntax: Follow this syntax which we could use in order to execute an asynchronous method async-await:

```
const submitScore = async (id, user, score) => {
  const response = await fetch(
    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`,
    {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
        Accept: 'application/json',
      },
      body: JSON.stringify({
        user,
        score,
      }),
    },
  );
  const data = await response.json();
  return data.result;
};
``` 
- For more documentation you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

Reply:
the promises are part of the project requirement as mentioned here:
<img width="448" alt="callbacks" src="https://user-images.githubusercontent.com/77294268/212458228-90301ef9-18af-4883-8c88-a0913d111122.png">